src/ptvsd/adapter/channels.py

```diff
@@ -28,13 +29,19 @@ class Channels(singleton.ThreadSafeSingleton):
     """
 
     @singleton.autolocked_method
-    def connect_to_ide(self):
+    def connect_to_ide(self, address=("localhost", None)):
```

Comment:
Let's do `address=None` instead for stdin/out. It's a bit confusing that the host gets specified, but then goes unused.